src/psmove.c

```diff
@@ -81,6 +87,30 @@
 #include "daemon/moved_client.h"
 #include "hidapi.h"
 
+#ifdef _MSC_VER
+#define STIN static __inline
+#else
+#define STIN static inline
+#endif
+
+#ifdef _MSC_VER
+#include "pstdint.h"
+static const unsigned __int64 epoch = 116444736000000000;
+int
+gettimeofday(struct timeval * tp, struct timezone * tzp)

```

Comment:
Did you implement this yourself or did you copy it from somewhere? If you copied it from somewhere, please mention the source in the comments and any license that might apply.